services/github/github-hacktoberfest.service.js

```diff
@@ -120,7 +120,14 @@ module.exports = class GithubHacktoberfestCombinedStatus extends GithubAuthV4Ser
       // We want to show "1 day left" on the last day so we add 1.
       daysLeft = moment('2019-11-01 12:00:00 Z').diff(moment(), 'days') + 1
     }
-
+    if (daysLeft <= 0) {
+      return {
+        message: `is over! (${metric(contributionCount)} ${maybePluralize(
```

Comment:
> `metric(contributionCount)`

So much optimism I had when I first wrote that 😀

Reply:
😄 Gotta supprot those `22Y PRs`

That does have me wondering which repos got the most PRs during Hacktoberfest

Reply:
I bet @MattIPv4 knows the answer to that. Though he may also be needing a big break right about now 😁

Reply:
Ask me again next week and I'll be able to get an answer for you. We don't have the full data yet as not everything finishes and settles down until the 8th :) (1st + 7 day review period)